src/print.rs

```diff
@@ -414,31 +415,36 @@ fn all_modules_uniq(module_list: &BTreeSet<String>) -> Vec<String> {
 fn load_formatter_and_modules<'a>(context: &'a Context) -> (StringFormatter<'a>, BTreeSet<String>) {
     let config = &context.root_config;
 
-    let lformatter = StringFormatter::new(&config.format);
-    let rformatter = StringFormatter::new(&config.right_format);
-    let cformatter = StringFormatter::new(&config.continuation_prompt);
-    if lformatter.is_err() {
-        log::error!("Error parsing `format`")
-    }
-    if rformatter.is_err() {
-        log::error!("Error parsing `right_format`")
-    }
-    if cformatter.is_err() {
-        log::error!("Error parsing `continuation_prompt`")
+    let (formatter, prompt_name) = match &context.target {
```

Comment:
Should this be `profile_name`? Or is `prompt_name` correct in this context?

Reply:
fixed